hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/KafkaOffsetGen.java

```diff
@@ -157,29 +161,23 @@ public static long totalNewMessages(OffsetRange[] ranges) {
 
     private static final String KAFKA_TOPIC_NAME = "hoodie.deltastreamer.source.kafka.topic";
     private static final String MAX_EVENTS_FROM_KAFKA_SOURCE_PROP = "hoodie.deltastreamer.kafka.source.maxEvents";
+    public static final String ENABLE_KAFKA_COMMIT_OFFSET = "hoodie.deltastreamer.source.enable.kafka.commit.offset";
```

Comment:
can we rename to `hoodie.deltastreamer.source.kafka.enable.commit.offset` to align with `hoodie.deltastreamer.source.kafka.topic` and would possibly rename `hoodie.deltastreamer.kafka.source.maxEvents` to `hoodie.deltastreamer.source.kafka.maxEvents`(should consider the compatibility)

Reply:
make sense, updated code, did not rename `hoodie.deltastreamer.kafka.source.maxEvents` in this PR, will discuss more on compatibility